requirements.txt

```diff
@@ -1,2 +1,3 @@
 diff_match_patch
-# additionally https://github.com/NVIDIA/NeMo is required
\ No newline at end of file
+nemo_toolkit[all]
```

Comment:
I think it's best not to add it directly in requirements.txt. There are some additional installation instructions listed in nemo repo which are important to follow. So it's best to just let the users know about that and ask to follow instructions in the nemo github